german-council-of-economic-experts.csl

```diff
@@ -293,6 +294,9 @@
       </if>
     </choose>
   </macro>
+  <macro name="doi">
+    <text variable="DOI" form="short" prefix=", doi:"/>
```

Comment:
do you have any input on the style format? DOIs should ideally be displayed as URLs (see e.g. [here](http://www.crossref.org/02publishers/doi_display_guidelines.html)), i.e. `prefix=", https://doi.org/"`. We'll take it like this if these are the style requirements and you can't change them, but I though I'd flag it.